mithril-aggregator/src/database/query/signed_entity/get_signed_entity.rs

```diff
@@ -60,6 +61,31 @@ impl GetSignedEntityRecordQuery {
             ),
         })
     }
+
+    pub fn by_signed_entity_type_and_epoch(
```

Comment:
It'd be better to create a `CardanoStakeDistribution` specific function here.

Reply:
Done ✅